### ISSUE TITLE
Level change in Half-Life causes a glitch sound effect

### ISSUE THREAD

**Issue by minihoot** [letsgoawaydev/gamesandstuff#10]
When I was doing the tram ride  at the beginning, I set host_framerate to 0.1
Then when the level changed, the LOUD sound of the door opening was just replaying over and over, and it was 1 segment, too.
Reproduction steps: (i think)
Go into Half-Life on gamesandstuff
Open the console (its the ` key) then do 'host_framerate 0.1'
Then wait for the level load while the  door is opening/closing.
This can also happen anywhere, however its worse with a loud sound effect.
Could also be with the Xash3D build.

**Comment by letsgoawaydev** [comment]
yeah this is a xash3d thing and also sometimes happens on half life steam

**Comment by Hoot679** [comment]
f

**Comment by minihoot** [comment]
This is so fucking old lol
